perception/autoware_image_projection_based_fusion/schema/fusion_common.schema.json

```
@@ -65,6 +65,21 @@
           "type": "number",
           "description": "Maximum z position [m].",
           "default": 100.0
+        },
```

`point_project_to_unrectified_image` parameter was missing from previous PR, could also add it?
```suggestion
        },
        "point_project_to_unrectified_image": {
          "type": "array",
          "description": "An array of options indicating whether to project point to unrectified image or not.",
          "default": [false, false, false, false, false, false]
        },
```

---

resolved https://github.com/autowarefoundation/autoware.universe/pull/9635/commits/3b0bbd0837696de640b608c17343789ee8fe70e5